presto-main/src/main/java/com/facebook/presto/sql/planner/PlanPrinter.java

```diff
@@ -418,38 +481,73 @@ private void printStats(int indent, PlanNodeId planNodeId, boolean printInput, b
             return;
         }
 
-        double fraction = (nodeStats.getWallTime().toMillis()) / (double) totalMillis;
-        String fractionString;
-        if (isFinite(fraction)) {
-            fractionString = format(Locale.US, "%.2f%%", 100.0 * fraction);
-        }
-        else {
-            fractionString = "?";
-        }
+        double fraction = 100.0d * nodeStats.getPlanNodeWallTime().toMillis() / totalMillis;
 
         output.append(indentString(indent));
-        output.append("Cost: " + fractionString);
+        output.append("Cost: " + formatDouble(fraction) + "%");
         if (printInput) {
             output.append(format(", Input: %s (%s)",
-                    formatPositions(nodeStats.getInputPositions()),
-                    nodeStats.getInputDataSize().toString()));
+                    formatPositions(nodeStats.getPlanNodeInputPositions()),
+                    nodeStats.getPlanNodeInputDataSize().toString()));
         }
         output.append(format(", Output: %s (%s)",
-                formatPositions(nodeStats.getOutputPositions()),
-                nodeStats.getOutputDataSize().toString()));
+                formatPositions(nodeStats.getPlanNodeOutputPositions()),
+                nodeStats.getPlanNodeOutputDataSize().toString()));
         if (printFiltered) {
-            double filtered = 100.0 * (nodeStats.getInputPositions() - nodeStats.getOutputPositions()) / nodeStats.getInputPositions();
-            String filteredString;
-            if (isFinite(filtered)) {
-                filteredString = format(Locale.US, "%.2f%%", filtered);
-            }
-            else {
-                filteredString = "?";
+            double filtered = 100.0d * (nodeStats.getPlanNodeInputPositions() - nodeStats.getPlanNodeOutputPositions()) / nodeStats.getPlanNodeInputPositions();
+            output.append(", Filtered: " + formatDouble(filtered) + "%");
+        }
+        output.append('\n');
+
+        printDistributions(indent, nodeStats);
+    }
+
+    private void printDistributions(int indent, PlanNodeStats nodeStats)
+    {
+        Map<String, Long> totalDrivers = nodeStats.getOperatorTotalDrivers();
```

Comment:
Take a look at example output of explain analyze with operators stats reporting:
```
 Fragment 2 [tpch:orders:1500000]                                                                                                                               
     Cost: CPU 7.54s, Input: 7501215 rows (128.77MB), Output: 4 rows (36B)                                                                                      
     Output layout: [count_6]                                                                                                                                   
     Output partitioning: SINGLE []                                                                                                                             
     - Aggregate(PARTIAL) => [count_6:bigint]                                                                                                                   
             Cost: 0.02%, Output: 4 rows (36B)                                                                                                                  
             AggregationOperator := Drivers: 4, Input avg.: 1500303.75 lines, Input std.dev.: 0.03%                                                             
             TaskOutputOperator := Drivers: 4, Input avg.: 1.00 lines, Input std.dev.: 0.00%                                                                    
             count_6 := "count"(*)                                                                                                                              
         - Project[] => []                                                                                                                                      
                 Cost: 0.38%, Input: 6001215 rows (206.04MB), Output: 6001215 rows (0B), Filtered: 0.00%                                                        
                 FilterAndProjectOperator := Drivers: 4, Input avg.: 1500303.75 lines, Input std.dev.: 0.03%                                                    
             - InnerJoin[("orderkey" = "orderkey_0")][$hashvalue, $hashvalue_7] => [orderkey:bigint, $hashvalue:bigint, orderkey_0:bigint, $hashvalue_7:bigint] 
                     Cost: 36.17%, Output: 6001215 rows (206.04MB)                                                                                              
                     LookupJoinOperator := Drivers: 4, Input avg.: 1500303.75 lines, Input std.dev.: 0.03%                                                      
                     HashBuilderOperator := Drivers: 16, Input avg.: 93750.00 lines, Input std.dev.: 0.23%                                                      
                                         := Collisions avg.: 117186.29 (99.60% est.), Collisions std.dev.: 1.56%                                                
                 - ScanProject[table = tpch:tpch:lineitem:sf1.0, originalConstraint = true] => [orderkey:bigint, $hashvalue:bigint]                             
                         Cost: 57.16%, Input: 6001215 rows (0B), Output: 6001215 rows (103.02MB), Filtered: 0.00%                                               
                         ScanFilterAndProjectOperator := Drivers: 4, Input avg.: 1500303.75 lines, Input std.dev.: 0.03%                                        
                         $hashvalue := "combine_hash"(BIGINT '0', COALESCE("$operator$hash_code"("orderkey"), 0))                                               
                         orderkey := tpch:orderkey                                                                                                              
                 - LocalExchange[HASH][$hashvalue_7] ("orderkey_0") => orderkey_0:bigint, $hashvalue_7:bigint                                                   
                         Cost: 6.19%, Output: 1500000 rows (25.75MB)                                                                                            
                         LocalExchangeSourceOperator := Drivers: 16, Input avg.: 93750.00 lines, Input std.dev.: 0.23%                                          
                         LocalExchangeSinkOperator := Drivers: 1, Input avg.: 1500000.00 lines, Input std.dev.: 0.00%                                           
                     - RemoteSource[3] => [orderkey_0:bigint, $hashvalue_8:bigint]                                                                              
                             Cost: 0.07%, Output: 1500000 rows (25.75MB)                                                                                        
                             ExchangeOperator := Drivers: 1, Input avg.: 1500000.00 lines, Input std.dev.: 0.00%      
```

`Aggregate`, `InnerJoin` and `LocalExchange` are all composed of multiple operators (half of total plan nodes). The information about skewness and hash collisions is especially useful for us when working on remote client cluster where we have limited access. In such cases by looking at `EXPLAIN ANALYZE` with individual operator stats we can quickly deduce how the data flow looks like and where there might be a bottleneck due to the skewness or excessive collisions. Since we have the information, it should be beneficial to display it even though one might not use every time (but rather when tracking some unexpected performance issue). Trying to squash the stats will cause some information to be hidden, which might hide valuable clues.